tests/mdp_test.py

```diff
@@ -0,0 +1,92 @@
+# This test includes unit tests for MarkovChain and MarkovDecisionProcess of
```

Comment:
It is a best practice (recommended by [PEP 257](https://www.python.org/dev/peps/pep-0257/)) to have a docstring at the start of a file instead of a comment block because this exposes the documentation to tools that read the `__doc__` attribute of modules. E.g., we do this in [transys_automata_test.py](https://github.com/tulip-control/tulip-control/blob/9b9a743d1486ded5a91eba2fbbb4536559b85f5f/tests/transys_automata_test.py).

However, I think this can be merged as-is and changed later because this is a file of tests, rather than a part of the `tulip` package.

Reply:
Thanks @slivingston. Sorry I didn't get a chance to address this review. I'll revisit these comments after the semester is over.